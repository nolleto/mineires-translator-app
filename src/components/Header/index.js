import React from 'react';

const Header = () => (
  <header className='header'>
    <h1>MineirÃªs Translator</h1>
  </header>
);

export default Header;
