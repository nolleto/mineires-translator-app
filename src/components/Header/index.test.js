import Header from './index';
import React from 'react';
import { shallow } from 'enzyme';

describe('Header component', () => {
  it('matches snapshot', () => {
    const snapshot = shallow(<Header />);

    expect(snapshot).toMatchSnapshot();
  });
});
