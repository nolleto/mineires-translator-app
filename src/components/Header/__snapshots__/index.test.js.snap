// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<header
  className="header"
>
  <h1>
    MineirÃªs Translator
  </h1>
</header>
`;
